cannot find module 'instructions.json'

I am creating a bundled angular app. I am using gulp and browserify to bundle the app. browserify has no problem bundling, but when the app loads in the browser, I get 
`Error: Cannot find module '/app/node_modules/loopback-boot/node_modules/instructions.json'.`

The stack error points to https://github.com/strongloop/loopback-boot/blob/master/browser.js#L20.
I have tried a couple of different combinations of the bundling script in https://github.com/strongloop/loopback-example-full-stack/blob/master/client/lbclient/build.js with no success. My next approach will to bundle the loopback client separately from the rest of the app. Any idea why this is not working correctly?

```js
//Gulp bundle function
function browserifyCommon() {
  var config;

  if (watching) {
    config = {
      cache: {},
      debug: true,
      packageCache: {},
      fullPaths: true
    };
  } else {
    config = {};
  }

  var b = browserify(config);

  try {
    boot.compileToBrowserify({
      appRootDir: paths.loopback
    }, b);
  } catch (err) {
    console.log('Loopback failed to bundle', err);
    process.exit(1);
  }

  if (watching) {
    console.log('Watching');
    b = watchify(b);
    b.on('update', function() {
      bundleItUp(b);
    });
  } 

  b.add(paths.appEntry);
  bundleItUp(b);
}

function bundleItUp(b) {
  console.log('Bundling');
  b.bundle()
    .pipe(plumber())
    .pipe(source('bundle.js'))
    .pipe(gulp.dest('/public'));
}
```

```js
//lbClient.js
var loopback = require('loopback'),
    boot = require('loopback-boot');

var app = loopback();
module.exports = app;
boot(app);
```

```js
//mainApp.js
require('./app.core');
require('./components');
require('./matView');

require('./loopback'); // Loopback required here! 

// # Main App entry point.
module.exports = angular
  .module('app', [
    'app.core',
    'app.templates',
    'app.mainView'
    //'app.loopback',
  ]);
```

--- a/docs/header-browser.md
+++ b/docs/header-browser.md
@@ -9,3 +9,7 @@ var boot = require('loopback-boot');
 var app = module.exports = loopback();
 boot(app);
 ```
+
+### Browserify Note
+
+Loopback-boot will *not work correctly* with `fullpaths` option set in browserify/watchify.

